library/src/main/scala/giter8/MavenHelper.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Original implementation (C) 2014-2015 Kenji Yoshida and contributors
+ * Adapted and extended in 2016 by foundweekends project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package giter8
+
+import scala.xml.{NodeSeq, XML}
+import org.apache.http.HttpResponse
+import org.apache.http.client.methods.HttpGet
+import org.apache.http.impl.client.DefaultHttpClient
+
+final case class StableVersion(major: Int, minor: Int, patch: Int) {
```

Comment:
I'd suggest we copy paste `VersionNumber` in sbt 0.13 that I wrote - https://github.com/sbt/sbt/blob/0.13/ivy/src/main/scala/sbt/VersionNumber.scala
In particular `toInt` is not safe since we can easily have version number with lots of digits.